src/libs/nav2_robot/src/ros_robot.cpp

```diff
@@ -23,6 +24,17 @@ RosRobot::RosRobot(rclcpp::Node * node)
 : node_(node), initial_pose_received_(false)
 {
   // Open and parser the URDF file
```

Comment:
Just noticed a typo here that you could fix while you're in this file ("parser" should be "parse").

Reply:
Fixed.